### PR TITLE
Add x11-ctrl module

### PR DESCRIPTION
**Pull request by IMbackK**
Turns x11-utils into a standalone module x11-ctrl, removes x11 fiddling from display* modules
Allows use of mce without/with no dependency on X

### CODE REVIEW

**Comment by IMbackK** [comment]
depends on https://github.com/maemo-leste/mce/pull/16

**Comment by IMbackK** [comment]
this pr is also, accidentally, a workaround for the d4 tklock rendering issue

the main point of this refactor was to alow mce to be build without x11 dependancy & to be used in a cli only environment

File: modules/inactivity-dev.c [hunk 1]
```diff
@@ -0,0 +1,439 @@
...
+static gint inactivity_timeout = DEFAULT_TIMEOUT;
+
+/**
+ * Enable/Disable blanking inhibit,
+ * based on charger status and inhibit mode
+ *
+ * @param timed_inhibit TRUE for timed inhibiting,
+ *                      FALSE for triggered inhibiting
+ */
+static bool inactiveity_inhibited(void)
```
Comment (MerlijnWajer) [inline modules/inactivity-dev.c]: inactivity*

**Comment by IMbackK** [comment]
this branch currently is showing the wrong commits for some reason, likey i messed up, ignore untill i fix

**Comment by IMbackK** [comment]
this pr now is correct and nolonger depends on #20

**Pull request merged**